docs/tutorials/join-dal-baker/verify-rights.md

```diff
@@ -61,6 +61,39 @@ Follow these steps to verify that your DAL node is receiving attestation rights:
 
          If this command returns a list of future attestation rights for your account, the delay has not expired yet and you must wait for that cycle to arrive.
 
+   - Check to see if you are active and re-register as a delegate if necessary:
+
+      1. Run this command to see if your account is active:
+
+         ```bash
+         octez-client rpc get /chains/main/blocks/head/context/delegates/$MY_BAKER | jq . | grep deactivated
+         ```
+
+         Baker accounts are deactivated when the baker is offline for a certain time.
+
+      1. If the value for the `deactivated` field is `true`, re-register as a baker by running this command:
+
+         ```bash
+         octez-client register key my_baker as delegate
+         ```
+
+      1. Find when the next cycle will start by going to a block explorer such as https://ghostnet.tzkt.io.
+      For example, this drop-down shows that the next cycle starts in 29 minutes:
+
+         ![The TZKT block explorer, showing information about the current cycle](/img/tutorials/tzkt-next-cycle.png)
+
+         When this cycle starts, Tezos calculates attestation rights for two cycles in the future and includes your baker.
```

Comment:
actually on mainnet `consensus_rights_delay` is 2 cycle,  but on ghostnet it is currently 3

Reply:
> When this cycle starts, Tezos calculates attestation rights for a certain number of cycles in the future and includes your baker. The number of cycles is the `consensus_rights_delay` network parameter